BrowserSelect/Form1.cs

```diff
@@ -104,9 +105,14 @@ public void add_rule(Browser b)
             }
             else
             {
-                // in case pattern was not ambiguous, just set the pattern as rule and open the url
-                var pat = (_alwaysRule.mode == 1) ? _alwaysRule.tld_rule : _alwaysRule.second_rule;
-                add_rule(b, pat);
+
+                _alwaysAsk = new ContextMenu((new[] { _alwaysRule.tld_rule, _alwaysRule.second_rule, _alwaysRule.url_rule })
+                    .Select(x => new MenuItem(x, (s, e) => add_rule(b, x))).ToArray());
+                // display the context menu at mouse position
+                _alwaysAsk.Show(this, PointToClient(Cursor.Position));
+             //   // in case pattern was not ambiguous, just set the pattern as rule and open the url
```

Comment:
The point of ambiguity detection was to hide the context menu (prevent the need for clicking multiple times) if not needed. The context menu was added for issue #13. Maybe make it show the context menu on right-click so the normal path of just adding the domain won't get longer requiring more clicks?